proto/catalog.proto

```diff
@@ -213,6 +213,8 @@ message Table {
   repeated int32 dist_key_in_pk = 25;
   // A dml fragment id corresponds to the table, used to decide where the dml statement is executed.
   optional uint32 dml_fragment_id = 26;
+  // The range of row count of the table.
+  plan_common.Cardinality cardinality = 27;
```

Comment:
nit: can you add some note like "It's optional" to avoid misuse of `unwrap` (and may be also "all new fields below are optional due to backward compatibility"

Reply:
Good point. Done